packages/rsocket-core/__tests__/RequestChannelStream.spec.ts

```diff
@@ -134,22 +127,13 @@ describe("RequestStreamStream Test", () => {
         expect(mockHandler.onComplete).toBeCalled();
         expect(mockHandler.onError).not.toBeCalled();
         expect(mockHandler.onNext).not.toBeCalled();
+        // TODO: why isn't `stream.disconnect()` called, thus removing the handler?
```

Comment:
Anywhere I added this `TODO` is where a test is currently failing due to `stream.disconnect()` not being called in the stream impl.